<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tux Chat Extreme</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="chat_tab">
       <attribute name="title">
        <string>Chat</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="QLabel" name="window_title">
            <property name="font">
             <font>
              <family>QDBetterComicSans</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>TUX CHAT EXTREME</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="host_box">
              <property name="font">
               <font>
                <family>QDBetterComicSans</family>
                <italic>false</italic>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <property name="placeholderText">
               <string>Server host</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="port_box">
              <property name="font">
               <font>
                <family>QDBetterComicSans</family>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Server port</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="username_box">
            <property name="font">
             <font>
              <family>QDBetterComicSans</family>
             </font>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="connect_button">
            <property name="font">
             <font>
              <family>QDBetterComicSans</family>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Wire it up!</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="window_divider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="chat_title">
            <property name="font">
             <font>
              <family>QDBetterComicSans</family>
             </font>
            </property>
            <property name="text">
             <string>Chat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="chat_box">
            <property name="font">
             <font>
              <family>QDBetterComicSans</family>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="input_box">
            <property name="font">
             <font>
              <family>QDBetterComicSans</family>
             </font>
            </property>
            <property name="placeholderText">
             <string>Enter text here and press enter to send</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="server_tab">
       <attribute name="title">
        <string>Server</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="server_title">
            <property name="font">
             <font>
              <family>QDBetterComicSans</family>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="text">
             <string>HOST A SERVER!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="server_port_box">
            <property name="font">
             <font>
              <family>QDBetterComicSans</family>
             </font>
            </property>
            <property name="placeholderText">
             <string>Port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="server_logs_title">
            <property name="font">
             <font>
              <family>QDBetterComicSans</family>
             </font>
            </property>
            <property name="text">
             <string>Logs:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="server_log_box">
            <property name="font">
             <font>
              <family>QDBetterComicSans</family>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="server_start_button">
            <property name="font">
             <font>
              <family>QDBetterComicSans</family>
             </font>
            </property>
            <property name="text">
             <string>Spin up a funky server</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
